meson.build

```diff
@@ -269,6 +269,12 @@ if get_option('plugin_thunderbolt')
   conf.set('HAVE_THUNDERBOLT', '1')
 endif
 
+if get_option('plugin_flashrom')
+  flashrom = find_program('flashrom', required: false)
+  libflashrom = cc.find_library('libflashrom', required: false)
```

Comment:
Two thoughts:
1. 'required' _should_ be set to true.  Basically if the option `plugin_flashrom` was turned on in the meson build configuration then that means that all the libraries it needs need to be installed too.
2. Is there a minimum version of `libflashrom` that has these needed interfaces?  It should be included here too.